packages/icon/scripts/build.ts

```diff
@@ -0,0 +1,177 @@
+const svgr = require('@svgr/core').default;
+// @ts-ignore
+const fs = require('fs');
+// @ts-ignore
+const template = require('../src/template');
```

Comment:
nit, put relative imports last

Reply:
👍 Think I glossed over this because we're using `require()`.